bin/cancel_squad_testjobs.py

```diff
@@ -46,7 +46,7 @@ def cancel_lava_jobs(
         project = squad_client.get_objects(base_url, params)[0]
     except:
         exit("Error: project {} not found at {}".format(project, base_url))
-    build_list = squad_client.get_objects(project["builds"], {"version": build_version})
+    build_list = squad_client.get_objects(project["builds"], {"version": build_version}, limit=50)
```

Comment:
alternatively, `class Builds()` is written as a generator so it will not suffer from this problem in the first place (what if build is 50 back?).

e.g. if you did "for build in Builds(project["builds"]):", it would not fetch _all_ builds, it would only fetch one page at a time. if you exited the loop at any time, the remaining builds would never be fetched.

Reply:
Pages are 50-builds long in SQUAD, so 50 is just one fetch. In `build_info.py` the default is `limit=10`, and for that SQUAD replies very very quickly.

I see the benefit of the generator (in 50-builds iterations, though), but what is the generator-y alternative?